src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-block: 32px 16px;
+  padding-inline: $padding-inline-value;
+  box-sizing: border-box;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $dark-blue-color-text;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    width: 98px;
+    color: $grey-color-text;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-text {
+    text-align: center;
+    font-size: 10px;
+    line-height: 14px;
+    color: $dark-blue-color-text;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    color: $grey-color-text;
+  }
+
+  &__price-value {
+    color: $dark-blue-color-text;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+
+    width: 100%;
+    height: 40px;
+    border: 1px solid $blue-color-button;
+    border-radius: 5px;
+    background-color: $blue-color-button;
+    color: $white-color-button;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    transition: $transition-duration;
+    background-color: $white-color-button;
+    color: $blue-color-button;
+  }
```

Need to check and fix it everywhere
```suggestion
  &__button {
    display: flex;
    justify-content: center;
    align-items: center;
    box-sizing: border-box;

    width: 100%;
    height: 40px;
    border: 1px solid $blue-color-button;
    border-radius: 5px;
    background-color: $blue-color-button;
    color: $white-color-button;
    font-size: 14px;
    font-weight: 700;
    line-height: 16px;
    text-transform: uppercase;
    text-decoration: none;
    
    &:hover {
      // write styles here
    }
  }
```